### PR TITLE
start namespace updates quotas pr

### PR DESCRIPTION
**Pull request by codyjlandstrom**

### CODE REVIEW

File: src/content/self-hosted/helm-configuration.mdx [hunk 1]
```diff
@@ -703,7 +703,7 @@ quickstarts:
 Enables [resource quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/) at the namespace level.
 
 - `resources`: Limits the number of resources that a user can create.
-  - `maxNamespaces`: Maximum number of namespaces.
+  - `maxNamespaces`: Maximum number of namespaces that a user can create.
```
Comment (pchico83) [inline src/content/self-hosted/helm-configuration.mdx:706]: I would document that this quota doesn't apply to `okteto-bot`. We can also mention it in the Admin tokens section

Comment (codyjlandstrom) [reply src/content/self-hosted/helm-configuration.mdx:706]: Should we wait until that work is complete or should we have it ready to ship for 1.19?

Comment (pchico83) [reply src/content/self-hosted/helm-configuration.mdx:706]: Oh yes, I was confused. Yes, we can merge this fix and wait for the other feature to be implemented

**Review by pchico83 — APPROVED** [review]

File: src/content/self-hosted/helm-configuration.mdx (hunk 1 above)
Comment (codyjlandstrom) [reply src/content/self-hosted/helm-configuration.mdx:706]: Looks like @ifbyol merged the PR. Making the update!

**Comment by codyjlandstrom** [comment]
Tagging @ifbyol to make sure this update is inline with DEV-249.

**Review by pchico83 — APPROVED** [review]

**Review by ifbyol — APPROVED** [review]
DEV-249 was merged and it will be out in the next release, so good to merge!

**Pull request merged**